packages/react-admin-core/src/finalFormRenderComponent.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { AnyObject, FormRenderProps, RenderableProps } from "react-final-form";
+
+// call to render-children consists of ...rest, ...renderProps, __versions, see https://github.com/final-form/react-final-form/blob/master/src/ReactFinalForm.js#L195
+// whereby ...rest is FormProps destructed with all Config attributes and FormProps specific attributes, see https://github.com/final-form/react-final-form/blob/master/src/ReactFinalForm.js#L42
+// which left RenderableProps in ...rest. (click FormProps and check extends). But they are missing in FormRenderProps-Type
+export type CorrectFormRenderProps<FormValues = AnyObject> = FormRenderProps<FormValues> & RenderableProps<FormRenderProps<FormValues>>;
```

Comment:
FormRenderProps are defined wrong in final-form-types package. this is a fix. (I'm going to create a pull-request in types)

Reply:
I addressed this missing props in this pull-request: https://github.com/final-form/react-final-form/pull/575
If my pull-request gets merged this type shouldn't be needed anymore.